### PR TITLE
Made it easier for people to find out what function to look for

### PR DESCRIPTION
**Pull request by Intoprelised**
I just added most names that I'm decently sure are correct. If some aren't correct pls tell me or change it.

### CODE REVIEW

**Comment by xDidntPot** [comment]
Nice.

**Comment by Synx173** [comment]
fr this is kinda needed

File: Memory/Hooks.cpp [hunk 1]
```diff
@@ -250,10 +250,10 @@ void Hooks::Init() {
 		// Mob::_jumpFromGround
 		void* jump = reinterpret_cast<void*>(FindSignature("48 89 5C 24 10 57 48 83 EC ?? 48 8B 05 ?? ?? ?? ?? 48 33 C4 48 89 44 24 ?? 48 8B 19 48 8D"));
 		g_Hooks.JumpPowerHook = std::make_unique<FuncHook>(jump, Hooks::JumpPower);
-
+		//ServerInstance::onAppSuspended (maybe)
```
Comment (Synx173) [inline Memory/Hooks.cpp:253]: below it says MinecraftGame and not serverinstance

File: Memory/Hooks.cpp [hunk 2]
```diff
@@ -976,6 +977,7 @@ void Hooks::Actor_lerpMotion(C_Entity* _this, vec3_t motVec) {
 
 		if (!networkSender) {
 			if (g_Data.getVersion() == GAMEVERSION::g_1_16_0)
+				// pretty sure its this Actor::lerpMotion
 				networkSender = reinterpret_cast<void*>(6 + FindSignature("FF 90 ?? ?? ?? ?? 4C 8D 9C 24 ?? ?? ?? ?? 49 8B 5B 18 49 8B 73 28 49 8B E3 5F C3"));
```
Comment (Synx173) [inline Memory/Hooks.cpp:981]: I think this has to do with network instead of lerpMotion (the hook is lerpmotion)

Comment (Intoprelised) [reply Memory/Hooks.cpp:981]: thanks, the devs can change it if they need to

File: Memory/Hooks.cpp (hunk 1 above)
Comment (Intoprelised) [reply Memory/Hooks.cpp:253]: thanks, the devs can change it if they need to

**Comment by Intoprelised** [comment]
> fr this is kinda needed

yeah

**Comment by Intoprelised** [comment]
gamer says unnecesary so i close :(

**Pull request closed without merging**